### PR TITLE
Feature/heads up display dev

### PR DESCRIPTION
**Pull request by Bmmaroni**
# Pull Request Template

Go from a screen that selects the game play of single player or multiplayer. 
When the single player mode is selected it goes into the screen which gives the option to read, write, and/or draw. 
    1. You will be able to check or uncheck draw or write but cannot choose both write and draw together
    2. After chooseing you will be able to upload the drawing or writing to the platform. 
When it is submited a function will pit the player against a randomly chosen boss. 
## Description
Add a summary that address the following:
 - motivation/purpose for the feature 
     To make a gamemode menu for single player where they can choose read write or draw and then upload their writing or drawing
 - What were those changes
     A gamemode menu for uploading a drawing by choosing the button. 
 - What issues were fixed
     A gamemode styled menu
 - Provide the link to the user story it is referencing in Trello board or other supporting document
     https://trello.com/c/Q4Wusyn7/53-create-button-to-select-single-player-mode

## Commits Checklist
- [x] commits have clear title names 
- [x] commit has descriptive message of what has changed 
- [x] more small commits than less big commits 

## Code Checklist 
- [x] code is formatted appropriately 
- [x] code meets DRY standards
- [x] no dead code (check for logs and print statements)  
- [x] no TODOS instead necessary comments 
- [x] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries   

### PRs needs to be reviewed by at least 2 team members
 - TPL is the 3rd reviewer 
 - Lots of comments and suggestions, please! 

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Comment by Bmmaroni** [comment]
[CSS/Animation dates for Progress bar](https://trello.com/c/CJeeiunL/80-css-animation-dates-for-progress-bar)

**Comment by Bmmaroni** [comment]
[Update state on checker in progress bar](https://trello.com/c/QFP9dzJk/82-update-state-on-checker-in-progress-bar)

**Comment by Bmmaroni** [comment]
[Add button and collapse](https://trello.com/c/oSjpUO2b/81-add-button-and-collapse)

File: src/components/pages/HeadsUpDisplay/CountdownTimer.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      year: "numeric",
+      hour: "numeric",
+      minute: "numeric",
+      second: "numeric"
+    });
+
+    let endTime = new Date();
+    let today = new Date().getDay();
+
+    console.log(today);
```
Comment (kwnie) [inline src/components/pages/HeadsUpDisplay/CountdownTimer.js]: Can we remove this console.log?

File: src/components/pages/HeadsUpDisplay/CountdownTimer.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  return (
+    <div className="countdown-timer">
+      {/* <div>
+        Current Activity: <CurrentActivity />
+      </div> */}
+      <div>Time Remaining: {convertTimeFormat(timeRemaining)}</div>
+      {/* <div>
+        Next Activity: <NextActivity />
+      </div> */}
```
Comment (kwnie) [inline src/components/pages/HeadsUpDisplay/CountdownTimer.js]: Do we need this commented out code?

File: src/styles/less/Hud.less [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+}
+.progbtn{
+    display: flex;
+    flex-direction: column;
+    font-size: var(--font-size);
+
+
+
+
+
```
Comment (kwnie) [inline src/styles/less/Hud.less]: We can delete this extra space

**Review by kwnie — CHANGES_REQUESTED** [review]
Good work!

File: src/components/pages/HeadsUpDisplay/CountdownTimer.js (hunk 1 above)
Comment (Bmmaroni) [reply src/components/pages/HeadsUpDisplay/CountdownTimer.js]: Absolutely! Good Catch thank you.

File: src/components/pages/HeadsUpDisplay/CountdownTimer.js (hunk 2 above)
Comment (Bmmaroni) [reply src/components/pages/HeadsUpDisplay/CountdownTimer.js]: Commented code will be removed yes

File: src/styles/less/Hud.less (hunk 3 above)
Comment (Bmmaroni) [reply src/styles/less/Hud.less]: Extra space is outta here!

**Review by kdneldor — DISMISSED** [review]
Looks good Brandon! I like the atomic commits. I would imagine that this is mergeable once the requested changes are made / conversations are resolved. Well done.

**Comment by PLisak777** [comment]
This all looks good to me, only suggestions I have would be removing a few unused declarations on your Hud.js file.
Line 1: useState is declared but never used
Line 3: Hud import is declared but never used.
Are you maybe saving these for later?

**Review by jproctor-rebecca — APPROVED** [review]
Thank you for resolving the merge conflicts and responding to the change requests so quickly.

**Review by TonySo88 — APPROVED** [review]
Looks great Noel! It was a pleasure working with you on this project.

**Review by kwnie — APPROVED** [review]
Great work! I like the comments, they clarify what is happening with the code.

**Review by kdneldor — APPROVED** [review]
Great work Brandon and team! I like the atomic commits. This looked like a lot of work.

**Review by bcanty92 — COMMENTED** [review]
Great work team! looks clean .It was a pleasure working with  everyone

**Pull request merged**